stix_shifter_modules/msatp/readme.md

```diff
@@ -0,0 +1,423 @@
+# MSATP - microsoft defender for endpoint module
```

Comment:
there's already a readme `stix-shifter/stix_shifter_modules/msatp/stix_translation/README.md`. we need to keep one readme. maybe update this one by combining relevant details from the older readme.

Reply:
thank you, I did not notice that. I merged mine with the original